### ISSUE TITLE
How to upload 10.000 images?

### ISSUE THREAD

**Issue by CarlosX99** [HashLips/solidity_smart_contracts#3]
I have follow the tutorial and I have been able to upload to the testnets.opensea.io a collection of 20 items but I cant do it with the entire collection.

When I try to change this line     mint(msg.sender, 20); to     mint(msg.sender, 10000); I got the following error:

![Captura](https://user-images.githubusercontent.com/90478260/133926034-45e2f0a8-9299-4fdb-ad5d-a7555415bb5e.PNG)

What I need to change to upload 10.000 images instead of only 20?

**Comment by BrokenSkullzNFT** [comment]
Trying to figure this out too

**Comment by BrokenSkullzNFT** [comment]
I'm trying with 1000 images and still doesn't work, well I don't know how to make it work.

**Comment by BrokenSkullzNFT** [comment]
DANIEL - HASHLIPS - Help us! Please ^_^

**Comment by Whatamidoingwrong1** [comment]
im trying to do 3333 images and it wont work either having the exact same error 

Daniel Please help us !!!!!

love your youtube content btw

**Comment by Clallenre** [comment]
Hi guys. Any solution to this issues? Any help would be appreciated. Attempting to upload 10thousand as well but hitting a road block with a similar error. Thanks in advance.
<img width="557" alt="Screen Shot 2021-09-28 at 1 29 18 PM" src="https://user-images.githubusercontent.com/33644707/135145157-e153e769-ca13-41a5-8eff-fb8245f6d2cd.png">
.

**Comment by cryptog4n9** [comment]
Also please instruct us on how to upload 1000 nfts a time on a collection of 10000 without the smart contract making a new collection? 

i have a colleciton of 10000 and i want to mint 1000nft per week into a single collection but i tried it and the next 1000 go to a new collection (collection name) + v2 v3 v4 and so on. thanks in advance! love your videos!

**Comment by sinanokatata** [comment]
> Have you tried editing this line? I think it would be helpful.
> 
> ![Screenshot_contract](https://user-images.githubusercontent.com/83508981/136652051-5b0f0c22-0351-46a0-8f7c-72777283ca7b.png)

is anyone tried this?

**Comment by sultan-710** [comment]
Yes i have tried that but no luck i have a collection of 2021. so u cant mint more then 200 in once. i tried a bunch of different numbers for me only number 200 worked. and now i mint 200 nft in every transction on polygon gas fee is like 0.58matic not that bad and it works smooth. just put the maxMintAmount to 200. thank you

**Comment by eugle** [comment]
Is it really impossible to cast 10000 NFTs? Now it is 2022. 10000 operations are very common. Who has a solution, or can other smart contracts do this? Please give me some advice. Thank you
